frontend/src/metabase/dashboard/containers/DashboardApp/DashboardApp.unit.spec.tsx

```diff
@@ -34,28 +34,19 @@ import {
 import { createMockEntitiesState } from "__support__/store";
 import { callMockEvent } from "__support__/events";
 
-const TEST_COLLECTION = createMockCollection();
-
-const TEST_DATABASE_WITH_ACTIONS = createMockDatabase({
-  settings: { "database-enable-actions": true },
-});
-
-const TEST_COLLECTION_ITEM = createMockCollectionItem({
-  collection: TEST_COLLECTION,
-  model: "dataset",
-});
-
-const TEST_CARD = createMockCard();
-
-const TEST_TABLE = createMockTable();
```

Comment:
Same cruft as everywhere else. We don't need "constants" when they're not exported or at least used multiple times, and absolutely certainly we don't need them at the top of the file — it's ugly and useless albeit quite common.